VRCFaceTracking/Params/Expressions/UnifiedExpressions.cs

```diff
@@ -0,0 +1,201 @@
+﻿namespace VRCFaceTracking.Params
+{
+    // TODO: Create expressions map to relate legacy shapes to new ones
+
+    /// <summary> 
+    /// Represents the type of Shape data being sent by UnifiedExpressionData, in the form of enumerated shapes.
+    /// </summary>
+    public enum UnifiedExpressions
+    {
+        #region Eye Gaze Expressions
+
+        // These are currently unused for expressions and used in the UnifiedEye structure.
+        // EyeLookOutRight,
+        // EyeLookInRight,
+        // EyeLookUpRight,
+        // EyeLookDownRight,
+        // EyeLookOutLeft,
+        // EyeLookInLeft,
+        // EyeLookUpLeft,
+        // EyeLookDownLeft,
+
+        #endregion
+
+        #region Eye Expressions
+
+        // 'Biometrically' accurate data that is included with UnifiedEye
+        //EyeClosedRight, // Closes the right eyelid
+        //EyeClosedLeft, // Closes the left eyelid
+        //EyeDilationRight, // Dilates the right eye's pupil
+        //EyeDilationLeft, // Dilates the left eye's pupil
+        //EyeConstrictRight, // Constricts the right eye's pupil
+        //EyeConstrictLeft, // Constricts the left eye's pupil
+
+        EyeSquintRight, // Squeezes the right eye socket muscles, causing the lower eyelid to constrict a little bit as well.
+        EyeSquintLeft, // Squeezes the left eye socket muscles, causing the lower eyelid to constrict a little bit as well.
+        EyeWideRight, // Right eyelid widens beyond the eyelid's relaxed position.
+        EyeWideLeft, // Left eyelid widens beyond the eyelid's relaxed position.
+
+        #endregion
+
+        #region EyeBrow Expressions
+
+        BrowInnerDownRight, // Inner right eyebrow pushes downward.
+        BrowInnerDownLeft, // Inner left eyebrow pushes downward.
+        BrowOuterDownRight, // Outer right eyebrow pushes downward.
+        BrowOuterDownLeft, // Outer Left eyebrow pushes downward.
+        BrowInnerUpRight, // Inner right eyebrow pushes upward.
+        BrowInnerUpLeft, // Inner left eyebrow pushes upward.
+        BrowOuterUpRight, // Outer right eyebrow pushes upward.
+        BrowOuterUpLeft, // Outer left eyebrow pushes upward.
+
+        #endregion
+
+        #region Nose Expressions
+
+        NoseDilationRight, // Right side nose's canal opens up.
+        NoseDilationLeft, // Left side nose's canal opens up.
+
+        NoseWrinkleRight, // Outer part of the right side nose pulls upward.
+        NoseWrinkleLeft, // Outer part of the right side nose pulls upward.
```

Comment:
NoseSneerLeft and NoseSneerRight

Reply:
All right sounds good! The initial name is from FAQS and this change would make it line up closer with the ARKit naming scheme.